packages/admin/admin-bundler/src/lib/plugin.ts

```diff
@@ -0,0 +1,57 @@
+import { readFileSync } from "fs"
+import { glob } from "glob"
+import path from "path"
+import { UserConfig } from "vite"
+
+export async function plugin() {
+  const vite = await import("vite")
+  const entries = await glob("src/admin/**/*.{ts,tsx,js,jsx}")
```

Comment:
NOTE: since node 22 we have the native glob API `https://nodejs.org/api/fs.html#fsglobpattern-options-callback` maybe we can wait since we also support node 20 😅 but I wanted to mention it for reference